libp2p/RLPxHandshake.cpp

```diff
@@ -246,6 +262,9 @@ void RLPXHandshake::cancel()
 
 void RLPXHandshake::error(boost::system::error_code _ech)
 {
+    if (m_originated)
```

Comment:
I think it's better to call this callback in any case (regardless of `m_originated`) and let Host decide whether it needs it.

(Because here it's not immediately clear why this condition exists)

Reply:
And I think you already ignore it if peer is not already known to Host, so we can just remove `if` here